build-aux/flatpak/org.qelectrotech.QElectroTech.json

```diff
@@ -0,0 +1,37 @@
+{
+  "id": "org.qelectrotech.QElectroTech",
```

Comment:
This might require some discussion.

Flatpak enforces a long-standing, not entirely written convention in the desktop space, which comes all the way from DBus, of having an "application id" formed like a reverse-DNS.

QElectroTech is maintained over at https://qelectrotech.org, so the obvious application id is `org.qelectrotech.QElectroTech`. We can change it to anything you like, but I'd **strongly** recommend to prefix it by `org.qelectrotech.`.

By convention, this application id corresponds to the name the application takes on DBus (if any), to the name of the desktop file, the appdata file, the icons, …

Flatpak actually enforces this convention, so the application id is important.

Reply:
Strongly agree. The AppStream metadata file should also be named accordingly.

Reply:
There is no appstream file for an app, the appstream file is at the repo level and concatenates all the individual app appdata/metainfo files.

If you meant the appdata file, I did say so in my comment already, and this is taken care of with the `rename-appdata` option which I detailed in another comment.